plutus-ir/test/TransformSpec.hs

```diff
@@ -17,12 +17,12 @@ transform = testNested "transform" [
 
 thunkRecursions :: TestNested
 thunkRecursions = testNested "thunkRecursions" [
-    goldenPir "listFold" (runQuote listFold),
-    goldenPir "monoMap" (runQuote monoMap)
+    goldenPir "listFold" listFold,
+    goldenPir "monoMap" monoMap
     ]
 
-listFold :: Quote (Term TyName Name ())
-listFold = thunkRecursionsTerm =<< do
+listFold :: Term TyName Name ()
+listFold = runQuote $  do
```

Comment:
Oops, stupid bug, noticed it in a test and wanted to emphasize it, but here it is. Fixing.

Reply:
Fixed.